experiment/autobumper/bumper/bumper.go

```diff
@@ -286,6 +287,21 @@ func Run(o *Options) error {
 			}
 		}
 
+		// Check to see if the proper fork exists and if it does not, create one.
```

Comment:
This doesn't consider the fact that a fork needs some time to be ready after being created. Also, we already have code for this problem, can we somehow re-use the logic of one for the other?

https://github.com/kubernetes/test-infra/blob/299a5a1244b05692963ae1380966ca76e9315f18/prow/external-plugins/cherrypicker/server.go#L521

Reply:
Done. I move most of the ensureFork logic to the github client so both cherrypicker and autobumper could use it. PTAL